packages/orn-locator/src/resolvers/books.ts

```diff
@@ -425,14 +426,15 @@ const doOpenSearch = async(searchClient: SearchClient, limit: number, q: string,
   return results.hits.hits.slice(0, limit);
 };
 
-const doLocateSearch = (query: string, limit: number) => (inputs: any[]): Promise<any[]> => {
+const doLocateSearch = (ornCacheStore: OrnCacheStore, query: string, limit: number) => (inputs: any[]): Promise<any[]> => {
   const getScore = parseSearchQuery(query);
```

Comment:
wait... didn't we made the search documents contain the orn response? do we have to re-resolve them after receiving the search results? can we use elasticsearch as the cache for books and pages?

Reply:
I forget why we didn't implement ElasticSearch for library search, but that's the only one that still does locate search...

Reply:
If we change the locate calls for books to hit ES that would make the remaining library locate logic trivial right?

Reply:
Looking at this, the search calls for books already hit ES (not locate calls though, those either hit the ORN api or are used by the ORN api itself to fetch the book, depending on what file they're in).

The library ORNs have a structure that's different from the book and page ORNs and are not currently indexed in open-search. We could probably create a new indexing strategy, i4, and index them though?

Let me know if I missed something.

Reply:
Oh to use ES as the cache for resolve as well instead of dynamo (if I understood this right)... uh we could probably do it, if we're OK with the ORNs never updating after the search cluster is created. Once again except for the libraries, unless we also implement a new indexing strategy for them.